config/locales/devise.en.yml

```diff
@@ -55,6 +55,7 @@ en:
       unlocked: "Your account has been unlocked successfully. Please sign in to continue."
   errors:
     messages:
+      authentication_failed: "Authentication failed"
```

Comment:
(minor) Sounds a bit too technical. On the other hand, #120 will outdated this change anyway ... :)